client/apis/volumegroupsnapshot/v1beta1/zz_generated.deepcopy.go

```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2024 The Kubernetes Authors.
+Copyright 2025 The Kubernetes Authors.
```

Comment:
Why are we updating the years in these files?
It was created earlier, and we are not changing anything in it apart from year.

Reply:
It is autogenerated. We would ideally have a CI that checks using `git diff` after generating its own set of files.